src/LibraryViewExtensionMSWebBrowser/LibraryViewController.cs

```diff
@@ -373,9 +368,14 @@ internal void AddLibraryView()
             browser.Loaded += Browser_Loaded;
             browser.SizeChanged += Browser_SizeChanged;
             LibraryViewController.SetupSearchModelEventsObserver(browser, dynamoViewModel.Model.SearchModel, this, this.customization);
-        }
 
+            browser.DpiChanged += Browser_DpiChanged;
```

Comment:
I think it would be a good idea to unsubscribe this event handler in the `Dispose` method below.